WalletWasabi.Tests/UnitTests/WabiSabi/Backend/Rounds/MultipartyTransactionStateTests.cs

```diff
@@ -98,13 +98,13 @@ public void MaxSuggestedSteppingTest()
 
 			var maxSuggested = round.Parameters.MaxSuggestedAmount;
 
-			if (!histogram.ContainsKey(maxSuggested))
+			if (!histogram.TryGetValue(maxSuggested, out int value))
 			{
 				histogram.Add(maxSuggested, 1);
 			}
 			else
 			{
-				histogram[maxSuggested]++;
+				histogram[maxSuggested] = ++value;
```

Comment:
Not sure about the change here. It's useless and might even be slightly less performing, but we are talking micro optimization here.

I'd personally prefer to stay with ContainsKey. If you want to go with TryGetValue I'd do:

```csharp
if (!histogram.TryGetValue(maxSuggested, out _))
{
	histogram.Add(maxSuggested, 1);
}
else
{
	histogram[maxSuggested]++;
}
```

Reply:
Done ae99412fb89ec196883c97740beb1ae608528470